main.js

```diff
@@ -0,0 +1,74 @@
+// grab the customer container
+const customerContainer = document.querySelector("#customers");
+
+// date conversion function
+function dateConvert(date) {
+  return moment(date).format("ll");
+}
+
+// loop through the customer objects and build html for each one
+for (let customer of customers) {
+  let card = document.createElement("div");
+  card.classList.add("card");
+  card.classList.add("small");
+  card.classList.add("col");
+  card.classList.add("s12");
+  card.classList.add("m9");
+  card.classList.add("l4");
+  // created a card for the customer
+
+  let thumb = document.createElement("div");
+  thumb.classList.add("card-image");
+  thumb.classList.add("circle");
+  let pic = document.createElement("img");
+  pic.src = customer.picture.large;
+
+  thumb.appendChild(pic);
+  card.appendChild(thumb);
+
+  // create a div for name and apply the materialize class to it
+  let name = document.createElement("div");
+  name.classList.add("card-title");
+  let firstName =
+    customer.name.first[0].toUpperCase() + customer.name.first.slice(1);
```

Comment:
This is another place where you could create a helper function to do this capitalization and minimize the repetition.